src/model/trust.page.model.ts

```diff
@@ -0,0 +1,27 @@
+type TrustDetails = {
```

Comment:
A question, for @markpit @mouhajer-ch @mwestacott @SteveHicksCH @dgolubevs @JoshFen95 ...

The current way that the rest of the app works is to have a model for the page that is also used for the session.
In this PR, there is a slight difference in that way of working, but there may be an argument for keeping this difference, as show below. Please share your thoughts.

We need a new page object: trust.page.model.ts
We could store this in the session and then alter the node sdk to cope with the second way of modelling trusts. The SDK would either use a feature flag, passed in, or look at the session and see one or other type of trust and use whichever one is present (erroring in the event that both are present, which should be impossible).

OR...
We could take the page model and map it to the existing trust.model.ts and store THAT in the session. In this case no changes are needed in the SDK.
At the end of the project, we can either:
keep it as is, only refactoring to get rid of the page that uses trust.model.ts, but continuing to use that as our session object.
OR
refactor to: 1) get rid of the trust.model.ts 2) use trust.page.model.ts as the session object, pass that into the SDK and 3) refactor the SDK.

The trade-off is between:
1) less work now, less churn now, with a little more work later to refactor
2) more work now (starting from this PR's starting point) and more churn with each milestone, but no need to refactor at the end.

Reply:
> refactor to: 1) get rid of the trust.model.ts 2) use trust.page.model.ts as the session object, pass that into the SDK and 3) refactor the SDK.

in this case it will work like now (

My approach separates a session object from a page object.
In current implementation session object used for page object and in other way

> refactoring
it is too much work and this is not cost it

Reply:
@markpit Would you be OK to leave a brief reply to this, in line with our chat earlier?

Reply:
yes ok, I think the simplest approach is to leave the API input data structure as it is, and the SDK should just act as a wrapper to the API. So that means the data in the session should be in the same structure as the API input data. It seems sensible to me to map the page data into the session/model structure that will be passed to API.

Reply:
After talking with @markpit , he says that he's OK with this as-is.